### DISCUSSION TITLE
Cobalt supports Option Lists?

### DISCUSSION THREAD

**Discussion by antonnymello** [Auties00/WhatsappWeb4j#423] [Q&A]
Currently, I have a WhatsApp API using Baileys (node.js). Baileys no longer supports the sending of option lists. I noticed that in the Cobalt documentation, the lists are not marked as "deprecated". I would like to inquire whether there is still support for lists or if the documentation is outdated.

**Comment by codespearhead** [comment]
Lists never worked properly in any unofficial WhatsApp API, so the documentation is outdated. Can you submit a PR?
